templates/docs.html

```diff
@@ -77,6 +77,7 @@
 	</ul>
 	{% endif %}
 </li>
+<li class="navleft-item"><a href="/docs/tools/">Tools</a></li>
```

Comment:
please use `url` template tag when referencing pages within cactus site.
`{% url '....html' %}`
this will warn you if the referenced page is not available
you can do a separate PR to change that in this template file, though.